docs/Glossary/index.md

```diff
@@ -0,0 +1,132 @@
+# Glossary
+
+This glossary provides definitions of commonly used terms and phrases, organized by application or function.
+
+---
+
+## Media Management Applications
+
+**Radarr**, **Sonarr**, and **Lidarr** are applications designed to manage your media libraries:
+
+- **Radarr**: For movies
+- **Sonarr**: For TV shows
+- **Lidarr**: For music
+
+### Custom Format
+
+A feature that matches specific aspects of a release based on information extracted from its name. Users choose the matching criteria, and Custom Formats can be assigned scores on a per-[Quality Profile](#quality-profile) basis.
```

Comment:
maybe add that the info could also come from metadata of the release? of course only after/on import

Reply:
> maybe add that the info could also come from metadata of the release? of course only after/on import

Technically it's still the name though, isn't it? The *arr will rename based on the metadata, and the CF matches the name?

Reply:
I think the technical distinction might confuse users, when we could simply note that the CF after import can change based on the file's actual properties rather than simply the name of the torrent on site.

Not all users are familiar with what the metadata of a file actually encompasses, and this could be ambiguous or completely miss users who are familiar with metadata being referred to for things like music which would be manually added versus actual quality/format/codec etc.

No real strong opinion here, just a consideration I wanted to raise. Seems like a more vague description that users familiar with metadata and mediainfo could understand while not convoluting the general public that may not be.

Reply:
> I think the technical distinction might confuse users, when we could simply note that the CF after import can change based on the file's actual properties rather than simply the name of the torrent on site.
> 
> Not all users are familiar with what the metadata of a file actually encompasses, and this could be ambiguous or completely miss users who are familiar with metadata being referred to for things like music which would be manually added or tags versus actual quality/format/codec etc.
> 
> No real strong opinion here, just a consideration I wanted to raise. Seems like a more vague description that users familiar with metadata and mediainfo could understand while not convoluting the general public that may not be.

I like this approach - if you fancy also having a stab at that during your review!

Reply:
Covered in zakary's updates